product_docs/docs/edbcloud/beta/getting_started/01_understanding_qotas_in_azure.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Understanding quotas in Azure
+Redirects:
+ - 01_check_resource_limits/e3
+---
+
+The following sections help you plan for quotas (service limits) and Azure Service Provider registrations required by your cluster.
+
+[Managing service limits and constraints in Azure](01a_managing_quotas) describes how to check that your clusters meet these quotas and how to raise your resource limits, if needed.
+
+### IP Address quota
+
+You need to increase the limit of `Public IP Addresses - Basic` and `Public IP Addresses - Standard` for the regions where you plan to deploy your clusters with the total number of clusters you plan to use. To understand the requirements, and recommended quotas, see the following sections.
+
+**Default quota**
+
+The default public IP address limits for Public IP Addresses Basic and Public IP Addresses Standards is set to 10. See [Public IP address limits](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#publicip-address) for more information.
+
+
+**Recommended quota**
+
+EDB recommends the following when requesting public IP address quota increases:
+
+* Public IP Addresses – Basic: number of clusters you plan to deploy in a given region
+* Public IP Addresses – Standard: number of clusters you plan to deploy in a given region
+
+### vCPU quota
+
+Any time a new VM is deployed the vCPUs for the VMs must not exceed the vCPU quota for the VM size family or the total regional vCPU quota. The VMs deployment fails if either of those quotas are exceeded. 
+
+!!! Note
+    The default number of total cores per subscription per region is 20. See [Virtual Machines limits - Azure Resource Manager](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#virtual-machines-limits---azure-resource-manager) for more information.
+
+In each region, EDB Cloud uses six ESv3 and six DSv2 virtual machine cores to manage your EDB Cloud infrastructure.
```

Comment:
This section is written a bit brittle-y:

1. We should probably not describe the clusters as requiring N Esv3 cores; we'll be offering other VM families in short order.
2. It's possible for customers to end up with multiple DPs; we need to be careful about how the first sentence is worded:

```suggestion
To launch BigAnimal in a region, BA requires six Esv3 [...]
```

Reply:
@ba-edb 
"We should probably not describe the clusters as requiring N Esv3 cores; we'll be offering other VM families in short order."

- I think I'll remove the whole N Esv3 explanation about number of cores? The user might not need to know this logic as long as we are giving an accurate recommendation on the number of required cores, correct?

- Should i list the Dv4 VM family in addition to the  ESv3 and six DSv2. Are there any more changes in with respect to the VM families used in BigAnimal?

- Who is the can help me with correct recommendations for the number of cores?